web/assets/css/dashboard.less

```diff
@@ -62,9 +62,8 @@
     background: @grey80;
     z-index: 2;
     opacity: 0.9;
-    padding-top: (35px * @scale);
-    padding-bottom: (35px * @scale);
-    padding-left: ((50px + @name-padding) * @scale);
+    padding-top: (64px * @scale);
```

Comment:
Could you get rid of this padding - it results in the padding being doubled for team headers after the first one, since there's padding under the cards, too:

<img width="303" alt="Screen Shot 2021-02-17 at 12 20 09 PM" src="https://user-images.githubusercontent.com/26583442/108242059-7c187c80-711a-11eb-97c6-1180a5f16f1a.png">

You'll also need to add a bottom margin to the `all pipelines` header (that's done in Elm)